rel/files/riak-cs-debug

```diff
@@ -368,6 +430,7 @@ if [ 1 -eq $get_syscmds ]; then
     [ -f /etc/security/limits.conf ] && dump limits.conf cat /etc/security/limits.conf
     [ -f /var/log/messages ] && dump messages cat /var/log/messages
     [ -f /var/log/syslog ] && dump messages cat /var/log/syslog
+    [ -f /var/log/kern.log ] && dump messages cat /var/log/kern.log
```

Comment:
Should the first arg for `dump` be anything other than `messages` to avoid concatenating files of different kind?
Not the diff of this PR, the line of `syslog` also uses `messages` .